library/errata_tool_product.py

```diff
@@ -327,6 +337,7 @@ def run_module():
         move_bugs_on_qe=dict(type='bool', default=False),
         text_only_advisories_require_dists=dict(type='bool', default=True),
         exd_org_group=dict(choices=list(EXD_ORG_GROUPS.keys())),
+        show_bug_package_mismatch_warning=dict(type='bool', default=True),
```

Comment:
Let's remove the `default` value here, for two reasons:
1. It's better to rely on the errata-rails developers to choose default values, and then errata-tool-ansible will not fall out of sync if the errata-rails defaults change in the future. 
2. More importantly, once this PR merges, for playbooks that omit this parameter, Ansible will begin to set this field to `true` on existing products where users may have set it to `false`.

(In hindsight I've set too many `default` values for parameters, and long-term I want to remove them, see https://github.com/ktdreyer/errata-tool-ansible/issues/129 for more explanation.)
```suggestion
        show_bug_package_mismatch_warning=dict(type='bool'),
```

It's fine to keep the `default: true` line in the documentation above, since that is just explaining the current errata-rails default behavior, not functionally altering existing products.

Reply:
good point, it's been removed: https://github.com/ktdreyer/errata-tool-ansible/pull/304/files#diff-fc39a95310d1ceca5885571789ee350fc3d83262c32b1c1e1f4ccf4103a9c7e4R340

Reply:
>  you can just add a line to tests/integration/errata_tool_product/create-1.yml

Added it: https://github.com/ktdreyer/errata-tool-ansible/pull/304/files#diff-8c0f329dcf9d40b453755b5da84bfb286b3f2e21aa81d0fecf2ffe0434a693cbR29